tsfeatures/tests/test_holt_parameters.py

```diff
@@ -1,18 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Please add these tests to nbs/tsfeatures.ipynb

Reply:
added the tests to the tsfeatures.ipynb